### PR TITLE
Updated maintainers' page.

### PR DESCRIPTION
**Pull request by bwplotka**
Signed-off-by: Bartek Plotka <bwplotka@gmail.com>

### CODE REVIEW

File: MAINTAINERS.md [hunk 1]
```diff
@@ -1,9 +1,11 @@
...
 
-| Name                | Email                 | GitHub                                   |
-|---------------------|-----------------------|------------------------------------------|
-| Bartłomiej Płotka   | bartek@improbable.io  | [@bwplotka](https://github.com/bwplotka) |
-| Dominic Green       | dom@improbable.io     | [@domgreen](https://github.com/domgreen) |
+| Name                | Email                 | Slack       | GitHub                                                     |
+|---------------------|-----------------------|-------------|------------------------------------------------------------|
+| Bartłomiej Płotka   | bartek@improbable.io  | `@bwplotka` | [@bwplotka](https://github.com/bwplotka)                   |
+| Dominic Green       | dom@improbable.io     | `@domgreen` | [@domgreen](https://github.com/domgreen)                   |
+| Frederic Branczyk   | ?                     | `@brancz`   | [@brancz](https://github.com/brancz)                       |
```
Comment (bwplotka) [inline MAINTAINERS.md]: @brancz I need your signature... here (: What's the email you want to use?

Comment (brancz) [reply MAINTAINERS.md]: fbranczyk@gmail.com

**Comment by GiedriusS** [comment]
Don't forget me! :smile: My e-mail is giedriuswork@gmail.com.

**Comment by brancz** [comment]
lgtm

**Pull request merged**